.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm: 2.4
+cache: bundler
+matrix:
+  include:
+    - os: linux
+    - os: osx
+    # - os: windows  --  windows doesn't have language: ruby support yet
```

Comment:
Simple travis file using a lot of Travis defaults, including `script: rake` on all platforms.  Windows could be added here, and probably will, but it wasn't as straightforward.